src/pyosmeta/contrib_review_meta.py

```diff
@@ -3,26 +3,40 @@
 related to peer review (Editors, Reviewers, Maintainers)
 """
 from dataclasses import dataclass
+from typing import Dict, List, Optional, Tuple, Union
 
 from .contributors import ProcessContributors
 
 
 @dataclass
 class UpdateReviewMeta(ProcessContributors):
-    def __init__(self):
+    """This is a class that is designed to add additional functionality
+    to ProcessContribs to support updating reviewer metadata associated with
+    review issues. It is a bridge between the ProcessContributors and
+    ProcessReviews objects."""
+
+    def __init__(self, GITHUB_TOKEN):
```

Comment:
if i inherit from another class what is the best way to inherit attrs such as the token?